Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -317,11 +311,11 @@ final class RecoveryTests: XCTestCase {
   func testRecovery24() {
     assertParse(
       """
-      while { true }() { //  expected-error 2 {{consecutive statements on a line must be separated by ';'}} expected-error {{closure expression is unused}} expected-note {{did you mean to use a 'do' statement?}} {{20-20=do }} expected-warning {{boolean literal is unused}}
+      while 1️⃣{ true }() { //  expected-error 2 {{consecutive statements on a line must be separated by ';'}} expected-error {{closure expression is unused}} expected-note {{did you mean to use a 'do' statement?}} {{20-20=do }} expected-warning {{boolean literal is unused}}
```

Comment:
While you're editing this, would you mind also removing the 
```
//  expected-error 2 {{consecutive statements on a line must be separated by ';'}} expected-error {{closure expression is unused}} expected-note {{did you mean to use a 'do' statement?}} {{20-20=do }} expected-warning {{boolean literal is unused}}
```

Looks like I forgot that when auditing the translated test cases.

Reply:
Like this?